src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -58,11 +58,11 @@ export const CourseCard = ({course, handleShowCourse, authorsList}) => {
 					<b>Created: </b>
 					<span>date</span>
 				</p>
-				<div>
+				<div className={styles.buttonsContainer}>
 
 					// reuse Button component for 'Show course' button
-					// reuse Button component for 'Delete' button with data-testid="deleteCourse"
-					// reuse Button component for 'Update' button with data-testid="updateCourse"
+					// reuse Button component with deleteButtonIcon from 'src/assets' for 'Delete' button with data-testid="deleteCourse"
```

Comment:
I think it would be great if icons were already imported to this file. Could you please add an import for icons

Reply:
added

Reply:
Added two proposals